build.gradle.kts

```
@@ -18,6 +18,7 @@ dependencies {
     implementation("com.google.api-client:google-api-client-gson:1.31.5")
 
     implementation("com.google.code.gson:gson:2.8.9")
+    implementation("org.conscrypt:conscrypt-openjdk-uber:2.5.2") // contains library code for all platforms
```

```suggestion
    implementation("org.conscrypt:conscrypt-openjdk-uber:2.5.2") {
        because("contains library code for all platforms")
    }
```

Adding the reason to `because` makes the reason visible to **consumers** of the library.